### ISSUE TITLE
[im2txt] Fine-tuning stage gives bigger loss than the initial-training stage

### ISSUE THREAD

**Issue by shaohuawan** [tensorflow/models#1370]
Training an im2txt model consists of two stages. In the first stage, the CNN weights are initialized from a pre-trained Inception-v3 model and fixed, and only the RNN weights are updated. In the second stage, the CNN and RNN weights are fine-tuned together.

I could reach a loss of around 0.4 after the first stage finishes, but the loss increases to around 0.8 after the second stage starts. The first-stage consists of 1,000,000 steps, as indicated in the README.

**The following is the log of the first-stage training.**
INFO:tensorflow:global step 999980: loss = 0.3836 (0.25 sec/step)
INFO:tensorflow:global step 999981: loss = 0.2878 (0.25 sec/step)
INFO:tensorflow:global step 999982: loss = 0.5148 (0.25 sec/step)
INFO:tensorflow:global step 999983: loss = 0.5335 (0.25 sec/step)
INFO:tensorflow:global step 999984: loss = 0.4786 (0.26 sec/step)
INFO:tensorflow:global step 999985: loss = 0.4112 (0.25 sec/step)
INFO:tensorflow:global step 999986: loss = 0.3929 (0.26 sec/step)
INFO:tensorflow:global step 999987: loss = 0.7933 (0.25 sec/step)
INFO:tensorflow:global step 999988: loss = 0.4637 (0.26 sec/step)
INFO:tensorflow:global step 999989: loss = 0.4626 (0.25 sec/step)
INFO:tensorflow:global step 999990: loss = 0.2558 (0.25 sec/step)
INFO:tensorflow:global step 999991: loss = 0.4778 (0.26 sec/step)
INFO:tensorflow:global step 999992: loss = 0.6048 (0.26 sec/step)
INFO:tensorflow:global step 999993: loss = 0.2212 (0.25 sec/step)
INFO:tensorflow:global step 999994: loss = 0.1857 (0.25 sec/step)
INFO:tensorflow:global step 999995: loss = 0.8599 (0.26 sec/step)
INFO:tensorflow:global step 999996: loss = 0.2753 (0.26 sec/step)
INFO:tensorflow:global step 999997: loss = 0.4142 (0.25 sec/step)
INFO:tensorflow:global step 999998: loss = 0.4301 (0.25 sec/step)
INFO:tensorflow:global step 999999: loss = 0.3907 (0.25 sec/step)
INFO:tensorflow:global step 1000000: loss = 0.3640 (0.25 sec/step)

**The following is the log of the second-stage training.**
INFO:tensorflow:global step 1000001: loss = 0.8442 (11.45 sec/step)
INFO:tensorflow:global step 1000002: loss = 0.7991 (1.37 sec/step)
INFO:tensorflow:global step 1000003: loss = 0.8291 (0.95 sec/step)
INFO:tensorflow:global step 1000004: loss = 0.8558 (0.81 sec/step)
INFO:tensorflow:global step 1000005: loss = 0.6748 (0.81 sec/step)
INFO:tensorflow:global step 1000006: loss = 1.2039 (0.81 sec/step)
INFO:tensorflow:global step 1000007: loss = 0.9719 (0.81 sec/step)
INFO:tensorflow:global step 1000008: loss = 1.1061 (0.81 sec/step)
INFO:tensorflow:global step 1000009: loss = 0.6259 (0.82 sec/step)
INFO:tensorflow:global step 1000010: loss = 0.7983 (0.82 sec/step)
INFO:tensorflow:global step 1000011: loss = 0.7596 (0.85 sec/step)
INFO:tensorflow:global step 1000012: loss = 1.0704 (0.93 sec/step)
INFO:tensorflow:global step 1000013: loss = 0.7508 (0.81 sec/step)
INFO:tensorflow:global step 1000014: loss = 1.0071 (0.85 sec/step)
INFO:tensorflow:global step 1000015: loss = 0.9809 (0.85 sec/step)
INFO:tensorflow:global step 1000016: loss = 0.7665 (0.85 sec/step)
INFO:tensorflow:global step 1000017: loss = 1.0815 (0.85 sec/step)
INFO:tensorflow:global step 1000018: loss = 1.0458 (0.85 sec/step)
INFO:tensorflow:global step 1000019: loss = 0.6666 (0.84 sec/step)
INFO:tensorflow:global step 1000020: loss = 0.9731 (0.85 sec/step)

**Comment by drpngx** [comment]
This question is better asked on  [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow) since it is not a  bug or feature request. There is also a larger community that reads questions there. Thanks!
